src/components/StepNavigation/StepNavigation.js

```diff
@@ -28,44 +34,65 @@ class StepNavigation extends React.Component {
   }
 
   render () {
-    const { activeStep, disabled, setStep, steps } = this.props
+    const { activeStep, disabled, setStep, showLabel, steps, totalPages } = this.props
     if (steps && steps.length > 1) {
       const nextStep = activeStep + 1
       const prevStep = activeStep - 1
       const options = steps.map((step, index) => {
         // We can't just use index for the value
         // because Grommet is using indexes internally as keys and this will error with a duplicate key
-        const value = `step-${index}`
+        const value = `step-${step}`
         return {
           disabled,
           id: value,
+          label: showLabel ? (step + 1).toString() : null,
           value
         }
       })
       return (
-        <Box direction='row'>
-          <StyledButton
+        <Box direction='row' gap='xsmall'>
+          <Button
+            data-index={0}
+            disabled={activeStep === 0}
+            icon={<FontAwesomeIcon icon={faAngleDoubleLeft} />}
+            onClick={() => setStep(0)}
+            plain
+          />
+          <Button
             data-index={prevStep}
             disabled={activeStep === 0}
-            icon={<FormPrevious />}
+            icon={<FontAwesomeIcon icon={faAngleLeft} />}
             onClick={() => setStep(prevStep)}
             plain
           />
-          <StyledRadioButtonGroup
-            direction='row'
-            gap='none'
-            name='step-selectors'
-            onChange={this.onChange.bind(this)}
-            options={options}
-            value={`step-${activeStep}`}
-          />
-          <StyledButton
+
+          <Box direction='row'>
+            {activeStep > 2 && <Text>...</Text>}
```

Comment:
[minor] For readability, this can be rewritten with `const hasMoreThan2Steps = activeStep > 2`